src/components/strategies/create/BuySellBlock/InputRange.tsx

```diff
@@ -16,6 +18,10 @@ export const InputRange: FC<{
   buy?: boolean;
   error?: string;
   setRangeError: (error: string) => void;
+  marketPricePercentages: {
+    min: BigNumber;
+    max: BigNumber;
```

Comment:
why not use the imported type `MarketPriceIndication`?

Reply:
You right, missed that. thanks